### ISSUE TITLE
Can't use reports option in CLI

### ISSUE THREAD

**Issue by robertlyson** [stryker-mutator/stryker-net#434]
Hi, 

Just getting this error when I try to run stryker with two reporters
```
➜  TestProject1 dotnet stryker --reporters ['ConsoleReport', 'ConsoleProgressBar']
zsh: bad pattern: [ConsoleReport,
➜  TestProject1
```
![image](https://user-images.githubusercontent.com/2392583/55340759-9a811a80-54a5-11e9-8f48-6f19e2203113.png)

I took this command from [docs](https://github.com/stryker-mutator/stryker-net/blob/master/src/Stryker.CLI/README.md#mutation-testrun-reporters).

System details:
- macOS 10.14.4 (18E226)
- dotnet version: 2.2.105

**Comment by Mobrockers** [comment]
Nice catch! The docs are wrong unfortunately. Because of the json syntax the value has to be enclosed in quotes.  Try `--reporters "['ConsoleReport', 'ConsoleProgressBar']"` instead. We'll be changing this to be more user friendly in the future.

**Comment by robertlyson** [comment]
Cool, thanks for pointing me into the good direction. Will send PR to fix the docs if you think that makes sense.

**Comment by Mobrockers** [comment]
Resolved by #435
